bfapi/service/jobs.py

```diff
@@ -208,11 +208,15 @@ def create(
         tide_max_24h=tide_max,
     )
 
-def __genCmd(
-        svc_format: str,
+# The services descended from pzsvc-exec all connect to a CLI (command-line interface) program.
```

Comment:
> This function is explicitly for the purpose of creating cmd strings to fit this particular case. If someone is making a change to this function that causes the comment to become inaccurate, that change itself is a mistake.

Any "safety" provided by this comment is an illusion.  The truth is, developers _will_ forget to update things like this when the underlying assumptions change.  With that in mind, there is a significantly higher probability that the function name will be changed before the comment will.

[Here's another perspective on this](http://softwareengineering.stackexchange.com/questions/254978/whats-wrong-with-comments-that-explain-complex-code).